examples/client/Locomotion/src/pages/Payments/index.js

```diff
@@ -45,14 +44,14 @@ export default ({ navigation, menuSide }) => {
     await usePayments.detachPaymentMethod(methodForDelete);
     await usePayments.loadCustomer();
     setLoading(false);
-    togglePopup('removeCard', false);
+    setIsCancelPopupVisible(false);
```

Comment:
because that is an outdated way we used to handle popups, its the only popup in that app still working through that context. either we should have them all that way or none

Reply:
So change the naming, it's not a cancel popup

Reply:
my bad, fixed